samples/k8s_v1_blueprint_invalid_with_nginx_dependency.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: k8s.cloudogu.com/v1
+kind: Blueprint
+metadata:
+  labels:
+    app.kubernetes.io/name: blueprint
+    app.kubernetes.io/instance: blueprint-sample
+    app.kubernetes.io/part-of: k8s-blueprint-operator
+    app.kubernetes.io/created-by: k8s-blueprint-operator
+  name: blueprint-sample
```

Comment:
Please set a unique name, so that blueprint resource will not be replaced.
Blueprints should stay in the cluster as a history, unchanged by the user after the initial apply.